ddl/reorg.go

```diff
@@ -220,7 +237,7 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, tblInfo *model.
 		case model.ActionModifyColumn:
 			metrics.GetBackfillProgressByLabel(metrics.LblModifyColumn).Set(100)
 		}
-		if err1 := t.RemoveDDLReorgHandle(job, reorgInfo.elements); err1 != nil {
+		if err1 := reorgInfo.CleanReorgMeta(); err1 != nil {
```

Comment:
This txn and DDL txn are not the same txn. There may be data and schema out of sync. I think this plan can be further optimized